app/views/moments/_form.html.erb

```diff
@@ -1,11 +1,6 @@
 <%= render partial: 'shared/editor_modal' %>
 
 <%= form_for(@moment) do |f| %>
-<% if @moment.errors.any? %>
```

Comment:
Similar comment made in the other PR, can't we put `<%= render 'validation_errors' %>` here?